kubernetes-membership-scheme/src/main/java/org/wso2/carbon/membership/scheme/kubernetes/KubernetesMembershipScheme.java

```diff
@@ -150,13 +154,16 @@ private Parameter getParameter(String name) {
             } catch (InterruptedException ignored) {
             }
             HazelcastUtil.sendMessagesToMember(messageBuffer, member, carbonCluster);
+            log.info(String.format("Current member list: %s", nwConfig.getJoin().getTcpIpConfig().setEnabled(true).getMembers()));
```

Comment:
Check log.isInfoEnabled as a string format operation is used inside log.info()

Reply:
Fixed with d72688e851a9b9f9f2a32888b2766eb42533046a